components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -5751,7 +5952,75 @@ def getBody(self):
 }
 %s
 vp.set(UndefinedValue());
-return true;""" % (getIndexedOrExpando, getNamed)
+return true;""" % (maybeCrossOriginGet, getIndexedOrExpando, getNamed)
+
+    def definition_body(self):
+        return CGGeneric(self.getBody())
+
+
+class CGDOMJSProxyHandler_set(CGAbstractExternMethod):
+    def __init__(self, descriptor):
+        args = [Argument('*mut JSContext', 'cx'), Argument('RawHandleObject', 'proxy'),
+                Argument('RawHandleId', 'id'), Argument('RawHandleValue', 'v'),
+                Argument('RawHandleValue', 'receiver'),
+                Argument('*mut ObjectOpResult', 'opresult')]
+        CGAbstractExternMethod.__init__(self, descriptor, "set", "bool", args)
+        self.descriptor = descriptor
+
+    def getBody(self):
```

Comment:
It looks like this function is the same for every proxy class that generates it. If that's the case, it could just be a common function in dom/bindings/proxyhandler.rs, rather than part of the codegen for each type.

Reply:
Addressed in 77012b72821fa197581789a32cda97f66b898c20